### PR TITLE
Forgot password and reset password feature

### PR DESCRIPTION
**Pull request by DystoriaX**

### CODE REVIEW

File: src/components/forget-password/ResetPassword.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    <Grid container component="main" className={classes.root}>
+      <CssBaseline />
+      <Grid item xs={false} sm={4} md={7} className={classes.image} />
+      <Grid item xs={12} sm={8} md={5} component={Paper} elevation={6} square>
+        <div className={classes.paper}>
+          <Avatar className={classes.avatar}>
+            <LockOutlinedIcon />
+          </Avatar>
+          <Typography component="h1" variant="h5">
+            FORGOT PASSWORD
```
Comment (richwill28) [inline src/components/forget-password/ResetPassword.js]: Shouldn't this be RESET PASSWORD?

**Review by richwill28 — APPROVED** [review]

File: src/components/forget-password/ResetPassword.js (hunk 1 above)
Comment (DystoriaX) [reply src/components/forget-password/ResetPassword.js]: Ah yes, thanks

File: src/components/ChangePassword.js [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+  form: {
+    width: "100%", // Fix IE 11 issue.
+    marginTop: theme.spacing(1),
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2),
+  },
+}));
+
+export default function ResetPassword() {
```
Comment (richwill28) [inline src/components/ChangePassword.js]: This should be <code>ChangePassword()</code>?

Comment (DystoriaX) [reply src/components/ChangePassword.js]: Ok

**Pull request merged**